test/unit/components/message.js

```diff
@@ -35,6 +35,10 @@ describe('<FormattedMessage>', () => {
         );
     });
 
+    it('should not throw when <IntlProvider> is missing from ancestry but has defaultMessage', () => {
```

Comment:
It would be clearer to update the description of the test above with "... and there is no defaultMessage".

Reply:
Addressed :)